### PR TITLE
Version/update

### PR DESCRIPTION
**Pull request by sjchin88**
!TODO - Complete the Sign Out function

### CODE REVIEW

File: tqa-web-app/src/app/dtos/v2/admin-auth.dto.v2.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        private expirationDate: Date
+      ) {}
+    
+    get adminEmail(){
+        return this.email;
+    }
+    get expireDate(){
+        return this.expirationDate;
+    }
+}
```
Comment (edwardlach) [inline tqa-web-app/src/app/dtos/v2/admin-auth.dto.v2.ts:22]: Not positive what this class is for?  You should be able to create an instance of AdminAuthResponseV2 just as easily and that would be the preferred pattern?

File: tqa-web-app/src/app/state/store/resources/adminauth/adminauth.effects.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        () => {
+          return this.actions$.pipe(
+            ofType(adminLoginSuccess),
+            tap((action) => {
+              this.router.navigate(['/']);
+            })
+          );
+        },
+        { dispatch: false }
+      );
```
Comment (edwardlach) [inline tqa-web-app/src/app/state/store/resources/adminauth/adminauth.effects.ts:48]: I would make sure the formatting of this matches across all effects.  Based on how we have done the rest, move the () => { up a line for all methods here and below.

File: tqa-web-app/src/app/state/store/resources/adminauth/adminauth.reducer.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    ...initialCRState
+};
+
+
+const onAdminLoginSuccess =     on<AdminAuthState, ActionCreator<string, Creator<any[], object>>[]>(
+    adminAuthActions.adminLoginSuccess,
+    (state: AdminAuthState, props:any): AdminAuthState =>({
+        ...state,
+        resource:props.admin,
+        loading:true
```
Comment (edwardlach) [inline tqa-web-app/src/app/state/store/resources/adminauth/adminauth.reducer.ts:20]: loading should be false on admin login success, the login process is no longer happening.  Loaded would be true, indicating that the login process has completed.

**Review by edwardlach — CHANGES_REQUESTED** [review]
Overall it looks really good!  A few small comments from me.

* There shouldn't be any files in version control from the .angular/cache file.  Those should be removed.

* I'm not sure what purpose the AdminV2 class is serving.  I think everything you are using it for can be completed with the AdminAuthResponseV2.  I would prefer to use that datatype throughout to more closely match the patterns we have used everywhere else.

* There were a few other small formatting changes that would be good to make

File: tqa-web-app/src/app/dtos/v2/admin-auth.dto.v2.ts (hunk 1 above)
Comment (sjchin88) [reply tqa-web-app/src/app/dtos/v2/admin-auth.dto.v2.ts:22]: adminauthresponseV2 take expiresIn in string, adminV2 turn the expiresIn into expirationDate: Date

File: tqa-web-app/src/app/state/store/resources/adminauth/adminauth.effects.ts (hunk 2 above)
Comment (sjchin88) [reply tqa-web-app/src/app/state/store/resources/adminauth/adminauth.effects.ts:48]: amended

File: tqa-web-app/src/app/state/store/resources/adminauth/adminauth.reducer.ts (hunk 3 above)
Comment (sjchin88) [reply tqa-web-app/src/app/state/store/resources/adminauth/adminauth.reducer.ts:20]: amended. One thing I don't understand, what is these two statuses are used for?

Comment (edwardlach) [reply tqa-web-app/src/app/state/store/resources/adminauth/adminauth.reducer.ts]: I only ended up using the statuses once, but they are to allow UIs to show real time status indicators.  When loading, you can show the user a spinner, once totally loaded you can remove the spinner.   Things like that.  You need some type of state changes to allow for that behavior.  I am using those state options when creating visitors if you want to see an example.

**Review by edwardlach — APPROVED** [review]

**Pull request merged**